src/core/config/type-cast/mappings.js

```diff
@@ -46,7 +49,13 @@ const mappings = {
     typeCaster: numberTypeCaster,
     defaultValue: defaultOptions.maxDisplayedTags,
   },
+  modelPropertyMacro: { typeCaster: nullableFunctionTypeCaster },
   oauth2RedirectUrl: { typeCaster: undefinedStringTypeCaster },
+  onComplete: { typeCaster: nullableFunctionTypeCaster },
```

Comment:
As we're using `nullableFunctionTypeCaster`, should we also update the default value to `null`?

Reply:
Yes please